CMakeLists.txt

```diff
@@ -211,16 +211,13 @@ endif()
 
 target_compile_definitions(snmalloc INTERFACE $<$<BOOL:CONST_QUALIFIED_MALLOC_USABLE_SIZE>:MALLOC_USABLE_SIZE_QUALIFIER=const>)
 
-# In debug and CI builds, link the backtrace library so that we can get stack
+# Link the backtrace library so that we can get stack
```

Comment:
@davidchisnall I am not sure if I should make this part of the change.  It is required to get it to print stack traces, but systems could use core dumps or debuggers to get those.